Source/WebCore/inspector/InspectorClient.h

```diff
@@ -44,6 +44,7 @@ enum class InspectorClientDeveloperPreference : uint8_t {
     PrivateClickMeasurementDebugModeEnabled,
     ITPDebugModeEnabled,
     MockCaptureDevicesEnabled,
+    NeedsSiteSpecificQuirks,
```

Comment:
Maybe `SiteSpecificQuirksEnabled` to match the other preferences here?

Reply:
Will do.